### ISSUE TITLE
It still doesn't work

### ISSUE THREAD

**Issue by WildrCard** [fcannizzaro/com.dim.streamdeck#29]
Unfortunately loadout are still a no go, and it appears other features are gone too

**Comment by Dillieo** [comment]
yea cant get the app to show a connection box or even how to for that matter

**Comment by delphiactual** [comment]
Are you using https://beta.destinyitemmanager.com ?

**Comment by GrootForever** [comment]
Ensure that you have removed the old chrome extension and that you are using https://beta.destinyitemmanager.com/

**Comment by WildrCard** [comment]
> Ensure that you have removed the old chrome extension and that you are using https://beta.destinyitemmanager.com/

yeah, it is not working at all

**Comment by WildrCard** [comment]
![da77d0fd665fe95725f9c174abd6e651](https://user-images.githubusercontent.com/107009805/179292564-d81ac914-27bc-4acb-8056-46ef34d6ab27.png)

**Comment by GrootForever** [comment]
When you press pick on DIM what happens on the dim instance in your browser?

**Comment by WildrCard** [comment]
> When you press pick on DIM what happens on the dim instance in your browser?

nothing happens, no changes at all

**Comment by LugalofKos** [comment]
Works completely fine for me now. Proof:
![image](https://user-images.githubusercontent.com/106779896/179293168-8d53aa52-3dc9-4525-bc2c-2c69d33f069f.png)
Couple things you need to do that i recall. i uninstalled and reinstalled the plugin jic. make sure in the beta DIM site you have the streamdeck setting enabled. then it should pop up a number on the streamdeck that you have to enter while selecting the loadout iirc, and after it verifies (kind of like a 2FA) it will pair your stream deck setting with DIM and when you click pick on DIM it will spawn a new window in DIM where your loadouts pop up and on the far right you do send to streamdeck or something like that. 

Sounds like if it isn't doing anything when you press pick on DIM, you may have messed up the numbers part where you cross verify. I accidentally skipped that first time through and i had the same issue as you

**Comment by GrootForever** [comment]
Yeah try all the steps that Lugal has suggested, does the metric button
update with crucible ranking etc or is it just the loadouts that are not
working?

**Comment by fcannizzaro** [comment]
ok so can you manually swap to this profile on your stream deck ? 
![image](https://user-images.githubusercontent.com/6225939/179300393-0a871b3c-1f43-4e32-803a-e20b4f8be222.png)

and do it again ?

**Comment by WildrCard** [comment]
> ok so can you manually swap to this profile on your stream deck ? ![image](https://user-images.githubusercontent.com/6225939/179300393-0a871b3c-1f43-4e32-803a-e20b4f8be222.png)
> 
> and do it again ?

![a47aa3d0238e6f10f2c1a68976e4a017](https://user-images.githubusercontent.com/107009805/179301166-84cc4b6d-3242-4606-a49f-c4ab17fe7fe1.png)

**Comment by fcannizzaro** [comment]
😕 I never saw this issue,  can you uninstall and install again the plugin from the store ?

**Comment by WildrCard** [comment]
> 😕 I never saw this issue, can you uninstall and install again the plugin from the store ?

Yes, I did that too a few times, :(

**Comment by fcannizzaro** [comment]
it a bit weird can you try to hit... meanwhile can you try tapping on these buttons ? (if it works and you are 🍀 you tap the right number...)

![image](https://user-images.githubusercontent.com/6225939/179303681-c23fc468-fa85-468c-b42a-a7024f57c2c5.png)

**Comment by keilunDev** [comment]
I have a streamdeck mini and I don't get any number to pick :(

**Comment by delphiactual** [comment]
Change profile on the stream deck app to `Enhanced Dim Read Only`

**Comment by stayupthetree** [comment]
I change the profile but no numbers show up. reinstalled everything, always a black screen. I can click on and it shows the 3 numbers, but choosing the correct one doesnt do anything
